x-pack/plugins/security_solution/public/actions/show_top_n/cell_action/show_top_n.tsx

```diff
@@ -42,12 +43,20 @@ export const createShowTopNCellActionFactory = createCellActionFactory(
   }): CellActionTemplate<SecurityCellAction> => ({
     type: SecurityCellActionType.SHOW_TOP_N,
     getIconType: () => ICON,
-    getDisplayName: ({ field }) => SHOW_TOP(field.name),
-    getDisplayNameTooltip: ({ field }) => SHOW_TOP(field.name),
-    isCompatible: async ({ field }) =>
-      fieldHasCellActions(field.name) &&
-      !UNSUPPORTED_FIELD_TYPES.includes(field.type) &&
-      !!field.aggregatable,
+    getDisplayName: ({ data }) => SHOW_TOP(data[0]?.field.name),
+    getDisplayNameTooltip: ({ data }) => SHOW_TOP(data[0]?.field.name),
+    isCompatible: async ({ data }) => {
+      const field = data[0]?.field;
+
+      return (
+        data.length === 1 && // TODO Add support for multiple values
```

Comment:
So far the lens attributes for rendering `Show top N` histogram only support single value. If we are going to support multiple values here, we'll have to update its lens attributes as well.

Reply:
So, it makes sense to keep this validation and never render `Show top N` for multiple values.

Reply:
Yeah, let's regard show top N doesn't compatible with multiple values

Reply:
Agree, if we ever add support for multiple `fields` in the "showTopN" action we'd need to pass the proper data, and also update the component props, the query, the chart, and so on. 
But I think there's no use case for it currently (multiple fields are only used in some table counters, they only have filter and timeline actions if I remember correctly), so using the first field in the data array is okay for now in this action. We can create a separate issue if we have to add support for it later on.